girder/web_client/src/collections/Collection.js

```diff
@@ -119,6 +132,21 @@ var Collection = Backbone.Collection.extend({
         return Math.ceil((this.offset - this.length) / this.pageLimit);
     },
 
+    /**
+     * Sets a specific pagenumber for loading by caluclating the offset
+     * @param {Number} pageNumber - the page that should be loaded based on the pageLimit size
+     * @param {Object} params - additional parameters to pass to the fetch call
+     * @returns {Promise} a fetch promise to retrieve more data
+     */
+    fetchPage: function (pageNumber, params) {
```

Comment:
Do you think it would be clearer to have the parameter passed to this function be the zero-based page number.  That way you would skip the subtraction here, and an addition elsewhere, but some other places would need to be adjusted in exchange?  If it stays this way, at least change the parameter docs to indicate that it is 1-based.